### DISCUSSION TITLE
why the imageLayer image deformation？

### DISCUSSION THREAD

**Discussion by indexlang** [mistic100/Photo-Sphere-Viewer#1250] [General]
when i add a image layer，the image deformation，why？

aaa.png：
![image](https://github.com/mistic100/Photo-Sphere-Viewer/assets/6028356/f4d98611-5f63-45bd-9fae-b59d6928492f)
result psv：
![image](https://github.com/mistic100/Photo-Sphere-Viewer/assets/6028356/9ad99209-5e26-4b7b-8da2-c0e761c277c3)

code：
` markers: [
                        {
                            orientation: "horizontal",
                            id: "marker-v64nrt1xzh",
                            imageLayer: "aaa.png",
                            position: [
                                { textureX: 7997, textureY: 5241 },
                                { textureX: 9164, textureY: 5209 },
                                { textureX: 9664, textureY: 5561 },
                                { textureX: 8125, textureY: 5864 },
                                // { yaw: 1.245, pitch: -0.345 },
                                // { yaw: 1.834, pitch: -0.198 },
                                // { yaw: 2.102, pitch: -0.384 },
                                // { yaw: 1.386, pitch: -0.64 },
                            ],
                        },
                    ],`

**Comment by mistic100** [comment]
See #1102

Also discussions are not for bug reports.
